### ISSUE TITLE
Query on Nemo models license

### ISSUE THREAD

**Issue by BSen007** [NVIDIA/NeMo#5037]
Do Nemo pre-trained models comes with commercial friendly license? I can see it comes with NGC. I did not see any restriction, but I want to be double sure on this.Thanks

**Comment by titu1994** [comment]
Most models released in NeMo ASR (usable with from_pretrained()) fall under CC-BY-4 license. This includes HF models for ASR. Other domains may have specific licenses that do not support commercial usage or are for academic use cases only (some TTS models, and some NMT models). Please always refer to the model card for appropriate license. 

There may be some NGC model cards with Huggingface equivalent. In that case, It is advised to use HF checkpoints and look at the license noted on HF, that same license applies to NGC. In all other cases, NGC license terms apply until we update model cards. Note that given we have some 70-80 model cards, it may take time to retroactively update so many cards.

**Comment by BSen007** [comment]
thanks for quick response!

**Comment by shanecarroll-smarsh** [comment]
Hi @titu1994 sorry but I have to bring this back up because there is a discrepancy.  

It seems like it's the intent to license most NeMo models as CC-BY unless otherwise stated; however, it seems like every ASR model *is* otherwise stated. E.g., from https://catalog.ngc.nvidia.com/orgs/nvidia/teams/nemo/models/stt_en_conformer_ctc_large:

> License to use this model is covered by the NGC [TERMS OF USE](https://ngc.nvidia.com/legal/terms) unless another License/Terms Of Use/EULA is clearly specified. By downloading the public and release version of the model, you accept the terms and conditions of the NGC [TERMS OF USE](https://ngc.nvidia.com/legal/terms).

And as mentioned in other discussions, these TOU contain some concerning wording, e.g.,

> ... you grant NVIDIA and its affiliates an irrevocable, perpetual, nonexclusive, worldwide license to make, have made, use, have used, reproduce, license, distribute, sublicense, transfer and otherwise commercialize Derivatives without any obligation or compensation to you ...

In other words, according to these TOU, NVIDIA can claim as their own any models I have fine-tuned and they are unambiguously not CC-BY.  

But on the HF hub that these models are explicitly CC-BY without any other TOU/license mentioned, which matches the non-legally-binding claim made in many of these discussions. See the same model as above: https://huggingface.co/nvidia/stt_en_conformer_ctc_large (CC-BY tag, no `License` section in the model card)

So,
* Are the NGC model cards wrong? 
* Is there another license attached to these models that I don't see? 
* Or is the use of these models bound by the NGC TOU?

Thanks
